docker/release/config/opensearch-dashboards/opensearch-dashboards-docker-entrypoint.sh

```diff
@@ -153,50 +153,70 @@ opensearch_dashboards_vars=(
     telemetry.sendUsageFrom
 )
 
-longopts=''
-for opensearch_dashboards_var in ${opensearch_dashboards_vars[*]}; do
-    # 'opensearch.hosts' -> 'OPENSEARCH_URL'
-    env_var=$(echo ${opensearch_dashboards_var^^} | tr . _)
+function setupSecurityDashboardsPlugin {
+    SECURITY_DASHBOARDS_PLUGIN="securityDashboards"
 
-    # Indirectly lookup env var values via the name of the var.
-    # REF: http://tldp.org/LDP/abs/html/bashver2.html#EX78
-    value=${!env_var}
-    if [[ -n $value ]]; then
-      longopt="--${opensearch_dashboards_var}=${value}"
-      longopts+=" ${longopt}"
+    if [ -d "$OPENSEARCH_DASHBOARDS_HOME/plugins/$SECURITY_DASHBOARDS_PLUGIN" ]; then
+        if [ "$DISABLE_SECURITY_DASHBOARDS_PLUGIN" = "true" ]; then
+            echo "Disabling OpenSearch Security Dashboards Plugin"
+            ./bin/opensearch-dashboards-plugin remove securityDashboards
+            cat $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml | sed "/^opensearch_security/d" | tee $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml
+            cat $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml | sed "s/https/http/g" | tee $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml
+        fi
     fi
-done
+}
 
-# Files created at run-time should be group-writable, for Openshift's sake.
-umask 0002
+function runOpensearchDashboards {
+    longopts=()
+    for opensearch_dashboards_var in ${opensearch_dashboards_vars[*]}; do
+        # 'opensearch.hosts' -> 'OPENSEARCH_URL'
+        env_var=$(echo ${opensearch_dashboards_var^^} | tr . _)
 
-##Security Dashboards Plugin
-SECURITY_DASHBOARDS_PLUGIN="securityDashboards"
-if [ -d "$OPENSEARCH_DASHBOARDS_HOME/plugins/$SECURITY_DASHBOARDS_PLUGIN" ]; then
+        # Indirectly lookup env var values via the name of the var.
+        # REF: http://tldp.org/LDP/abs/html/bashver2.html#EX78
+        value=${!env_var}
+        if [[ -n $value ]]; then
+            longopt="--${opensearch_dashboards_var}=${value}"
+            longopts+=("${longopt}")
+        fi
+    done
 
-    if [ "$DISABLE_SECURITY_DASHBOARDS_PLUGIN" = "true" ]; then
-        echo "Disabling OpenSearch Security Dashboards Plugin"
-        ./bin/opensearch-dashboards-plugin remove securityDashboards
-        cat $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml | sed "/^opensearch_security/d" | tee $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml
-        cat $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml | sed "s/https/http/g" | tee $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml
-    fi
-fi
+    # Files created at run-time should be group-writable, for Openshift's sake.
+    umask 0002
+
+    ##Security Dashboards Plugin
+    setupSecurityDashboardsPlugin
 
+    # TO DO:
+    # Confirm with Mihir if this is necessary
```

Comment:
Let's figure this one out and remove/update. @mihirsoni ?

Reply:
@dblock We need to test with the latest version of linux and see if we really need this or not.

Reply:
Care to open an issue describing what this is about ?